src/rard/templates/research/partials/ordered_work_area.html

```diff
@@ -105,3 +109,43 @@
 </div>
 
 {% endwith %}
+
+<script>
+    document.addEventListener('DOMContentLoaded', (event) => {
```

Comment:
A simpler approach would have been to embrace the jQuery:

```
const collapsibleElements = $('.collapse');
collapsibleElements.on('show.bs.collapse', (event) => {
    var btn = $(event.target).parent().find('.collapse-button');
   // etc
}
collapsibleElements.on('hide.bs.collapse', (event) => {
  // etc
}
```

Reply:
The "Expand All" button also needs to have it's contents toggled to "Collapse All" if clicked.

Reply:
I had tried it with the event listener but couldn't get it to work the way I did it (with addEventListener), does this way work? Also if it does, if you open one section and not another and then click expand all, does it toggle them or does it make them all open?

Reply:
It appears that it's a simple toggle so if some sections are already open when a user clicks "Expand All", they would close while the other sections open.

A way to fix that might be to write a custom function to add the "show" class to all the collapsible sections rather than relying on Bootstrap's multi-collapse approach